docs/features/autoupdater.rst

```diff
@@ -8,10 +8,10 @@ Building Images
 
 By default, the autoupdater is disabled (as it is usually not helpful to have unexpected updates
 during development), but it can be enabled by setting the variable GLUON_BRANCH when building
-to override the default branch set in the set in the site configuration.
+to override the default branch set in the site configuration.
```

Comment:
maybe we can formulate this differently to make clear, that the value set in the site.conf is NEVER used at all. either you override it with GLUON_BRANCH or the autoupdater is disabled, thus doesn't use the value set  in the site.conf either.

Reply:
Which default branch is set building an image without `GLUON_BRANCH`?

Reply:
the branch set in site.conf is set, but the autoupdater will be disabled too, so the branch that is set is not used unless you re-enable the autoupdater by hand. I think this is not a very good solution right now, because there is no disable setting in site.conf. You use the fact that GLUON_BRANCH is missing during build as "switch" to disable the autoupdater.

Anyway as long as it is like it is, we should explain this here, so someone new will understand this behaviour.